src/ARKit/ARSkeleton.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+
+using Foundation;
+using ObjCRuntime;
+
+namespace ARKit {
+	public partial class ARSkeleton {
+
+		[iOS (14,0)]
+		[DllImport (Constants.ARKitLibrary)]
+		static extern IntPtr /* NSString */ ARSkeletonJointNameForRecognizedPointKey (/* NSString */ IntPtr recognizedPointKey);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
```

Comment:
Use the enum from Visio or use the smart enum ARSkeletonJointName rather than create one from the key

Reply:
I get that :-) but I mean what's the method/API alternative that the IDE won't try to hide from the developer ?

We normally use `Advanced` because there's a simpler/nicer alternative that we want developers to use (over the more complex one).